ext/hal/st/lib/sensor/vl53l0x/api/platform/src/vl53l0x_platform.c

```diff
@@ -147,44 +154,51 @@ VL53L0X_Error VL53L0X_ReadMulti(VL53L0X_DEV Dev, uint8_t index, uint8_t *pdata,
 VL53L0X_Error VL53L0X_WrByte(VL53L0X_DEV Dev, uint8_t index, uint8_t data){
     VL53L0X_Error Status = VL53L0X_ERROR_NONE;
     int32_t status_int;
-	uint8_t deviceAddress;
-
-    deviceAddress = Dev->I2cDevAddr;
 
-	status_int = VL53L0X_write_byte(deviceAddress, index, data);
+    status_int = i2c_reg_write_byte(Dev->i2c, Dev->I2cDevAddr, index, data);
 
-	if (status_int != 0)
-		Status = VL53L0X_ERROR_CONTROL_INTERFACE;
+    if (status_int < 0){
+        Status = VL53L0X_ERROR_CONTROL_INTERFACE;
+        SYS_LOG_ERR("i2c_reg_write_byte failed (%d)", Status);
+    }
 

```

Comment:
why modify all this, why not just provide a version of VL53L0X_write_byte?

Reply:
same as previous remark